src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -88,7 +89,7 @@
 @Component
 public class AzureProvider implements CloudProvider {
 
-    private final Logger logger = Logger.getLogger(AzureProvider.class);
+    private static final Logger logger = Logger.getLogger(AzureProvider.class);
 
     @Getter
     private final String type = "azure";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Make this final field static too. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1170)